openedx/core/djangoapps/content/learning_sequences/models.py

```diff
@@ -67,6 +67,10 @@ class Meta:
         ]
 
 
+class CourseContext(LearningContext):
```

Comment:
@ahmedbilal, nit: missing docstring. Also, is there a reason why you want to use concrete inheritance instead of abstract inheritance or composition?

Reply:
Done. Foreign keys also directed to `CourseSection` for models specific to course.